### ISSUE TITLE
Series true division with integers causing GDF_UNSUPPORTED_DTYPE error

### ISSUE THREAD

**Issue by beckernick** [rapidsai/cudf#217]
- [x] I am using the latest version of PyGDF built from `master`.
- [x] I have included the following environment details:
      Linux Distro, Linux Kernel, GPU Model
- [x] I have included the following version information for:
      Arrow, CUDA, Numpy, Pandas, Python
- [x] I have included below a minimal working reproducer (if you are unsure how
 to write one see http://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports).

```
VERSION="16.04.5 LTS (Xenial Xerus)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 16.04.5 LTS"
VERSION_ID="16.04"
HOME_URL="http://www.ubuntu.com/"
SUPPORT_URL="http://help.ubuntu.com/"
BUG_REPORT_URL="http://bugs.launchpad.net/ubuntu/"
VERSION_CODENAME=xenial
UBUNTU_CODENAME=xenial


Operating System:
VERSION="16.04.5 LTS (Xenial Xerus)"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 16.04.5 LTS"
VERSION_ID="16.04"

GPU:  Tesla V100

CUDA 9.2

Python 3.6.6

Arrow: N/A
pandas==0.20.3
numpy==1.14.3
```

When using the standard Dockerfile from `master`, standard true division on a `Series` of dtype `int`, `np.int32`, and `np.int64` is unexpectedly resulting in a `GDFError: GDF_UNSUPPORTED_DTYPE`, with the example code and traceback shown below:

```
import numpy as np
import pandas as pd
from pygdf import Series, DataFrame

s = Series(pd.Series(np.random.sample(10), dtype=int))
s / 2
```
```
---------------------------------------------------------------------------
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/conda/envs/gdf/lib/python3.6/site-packages/pygdf-0.1.0a3+24.gc6ee1af-py3.6.egg/pygdf/series.py", line 294, in __truediv__
    return self._binaryop(other, 'truediv')
  File "/conda/envs/gdf/lib/python3.6/site-packages/pygdf-0.1.0a3+24.gc6ee1af-py3.6.egg/pygdf/series.py", line 269, in _binaryop
    outcol = self._column.binary_operator(fn, other._column)
  File "/conda/envs/gdf/lib/python3.6/site-packages/pygdf-0.1.0a3+24.gc6ee1af-py3.6.egg/pygdf/numerical.py", line 83, in binary_operator
    out_dtype=lhs.dtype)
  File "/conda/envs/gdf/lib/python3.6/site-packages/pygdf-0.1.0a3+24.gc6ee1af-py3.6.egg/pygdf/numerical.py", line 349, in numeric_column_binop
    null_count = _gdf.apply_binaryop(op, lhs, rhs, out)
  File "/conda/envs/gdf/lib/python3.6/site-packages/pygdf-0.1.0a3+24.gc6ee1af-py3.6.egg/pygdf/_gdf.py", line 71, in apply_binaryop
    binop(*args)
  File "/conda/envs/gdf/lib/python3.6/site-packages/libgdf_cffi-0.1.0a1-py3.6.egg/libgdf_cffi/wrapper.py", line 27, in wrap
libgdf_cffi.wrapper.GDFError: GDF_UNSUPPORTED_DTYPE
```

This error only appears to occur with `int` dtypes (both numpy as standard Python) and only with true division (addition/subtraction/multiplication/floor division still work). It appears to be coming from an error in libgdf exposed through the wrapper in `def __getattr__` within  `libgdf`'s `python/libgdf_cffi/wrapper.py`.

**Comment by kkraus14** [comment]
@beckernick thanks for the issue! Looks like this is due to libgdf only having elementwise division operations for float32 and float64 dtypes: https://github.com/gpuopenanalytics/libgdf/blob/master/src/binaryops.cu#L229. Could you raise an issue against libgdf for supporting additional types as well? This shouldn't block #213 for now though.

**Comment by beckernick** [comment]
Will do.

**Comment by kkraus14** [comment]
Fixed by #612.
